### ISSUE TITLE
RangeSelector button hover state gets destroyed when adding data to a series

### ISSUE THREAD

**Issue by klaarseSICKAG** [highcharts/highcharts#19851]
#### Expected behaviour
The hover state of the RangeSelector buttons should not be influenced if data is added to a series in the background.

#### Actual behaviour
The hover state is not visible if the cursor hovers over a RangeSelector button while  data is added to a series.

#### Live demo with steps to reproduce
[](https://jsfiddle.net/stormtroopa/9v6n8z35/2/)

#### Product version
11.1.0

#### Affected browser(s)
Chrome

**Comment by bm64** [comment]
Hello @klaarseSICKAG 

Just to make sure we're on the same page, can you confirm that this demo correctly recreates the issue? 
 https://jsfiddle.net/BlackLabel/8g74qkze/. Do I understand correctly that the issue is the hover effect disappearing after data is set for the series?

**Comment by klaarseSICKAG** [comment]
Yes it does. I updated my initial post, so my demo is now visible. It shoes the effect more drastically.

**Comment by bm64** [comment]
@klaarseSICKAG thanks for adding the demo, I'm labeling the ticket as a bug.
